{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "children": [],
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:ibm:datapower_gateway:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndIncluding": "10.0.1.9",
            "versionStartIncluding": "10.0.1.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:ibm:datapower_gateway:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndIncluding": "10.0.4.0",
            "versionStartIncluding": "10.0.3.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:ibm:datapower_gateway:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndIncluding": "2018.4.1.22",
            "versionStartIncluding": "2018.4.1.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:ibm:datapower_gateway:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndIncluding": "10.5.0.2",
            "versionStartIncluding": "10.5.0.0",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "psirt@us.ibm.com",
      "ID": "CVE-2022-40228"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM DataPower Gateway 10.0.3.0 through 10.0.4.0, 10.0.1.0 through 10.0.1.9, 2018.4.1.0 through 2018.4.1.22, and 10.5.0.0 through 10.5.0.2 does not invalidate session after a password change which could allow an authenticated user to impersonate another user on the system. IBM X-Force ID: 235527."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-613"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://www.ibm.com/support/pages/node/6840759",
          "refsource": "MISC",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://www.ibm.com/support/pages/node/6840759"
        },
        {
          "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/235527",
          "refsource": "MISC",
          "tags": [
            "VDB Entry",
            "Vendor Advisory"
          ],
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/235527"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV3": {
      "cvssV3": {
        "attackComplexity": "LOW",
        "attackVector": "NETWORK",
        "availabilityImpact": "NONE",
        "baseScore": 5.4,
        "baseSeverity": "MEDIUM",
        "confidentialityImpact": "LOW",
        "integrityImpact": "LOW",
        "privilegesRequired": "LOW",
        "scope": "UNCHANGED",
        "userInteraction": "NONE",
        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:L/A:N",
        "version": "3.1"
      },
      "exploitabilityScore": 2.8,
      "impactScore": 2.5
    }
  },
  "lastModifiedDate": "2022-11-26T03:30Z",
  "publishedDate": "2022-11-22T19:15Z"
}